hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -87,7 +87,30 @@ private Relationships() {
          * <p>When deleting a resource, all its contained child resources are deleted along with it as mandated by the
          * contract of the {@link #contains} relationship.
          */
-        isParentOf
+        isParentOf,
+
+        /**
+         * A relationship from a resource to a structural data that contains the configuration of that resource.
+         *
+         * <p>The relationship is 1-to-1. There is no sharing of configuration objects between different resources.
```

Comment:
`0-1 to 0-1` right? Configuration can live without a resource and vice-versa. I can imagine that there will be a lot of leaves in the graph (resource nodes) without any configuration and the more abstract ones (higher in the resrouce hierarchy) will have the configuration. Perhaps some kind of following of the resource hierarchy could make a sense in the future + the ability to override it in the children.